understanding/20/use-of-color.html

```diff
@@ -63,6 +63,16 @@ <h2>Intent of Use of Color</h2>
 
       </div>
 
+     <div class="note">
+       
+       <p>This criterion is not applicable when assessing adjacent content that is rendered as the same color. For instance,
```

Comment:
i'd go even broader to start with: `this criterion does not apply to situations where colour has *not* been used to convey information, indicating an action, prompting a response, or distinguishing a visual element.` and then move into the more specific example/case

Reply:
pulled in your suggestion.  thanks